style.css

```diff
@@ -0,0 +1,6 @@
+.head {
+  font-style: normal;
+  font-size: large;
+  text-align: center;
+  font-weight: bold;
```

Comment:
Please remember to add colour to your style as requested in the [requirement](https://microverse.pathwright.com/library/new-technical-curriculum/177956/path/step/104219204/)